redigomock.go

```diff
@@ -182,12 +182,15 @@ func (c *Conn) do(commandName string, args ...interface{}) (reply interface{}, e
 	c.stats[cmd.hash()]++
 	c.statsMut.Unlock()
 
+	cmd.Called = true
 	if len(cmd.Responses) == 0 {
 		return nil, nil
 	}
 
 	response := cmd.Responses[0]
-	cmd.Responses = cmd.Responses[1:]
+	if len(cmd.Responses) > 1 {
```

Comment:
Don't need to use this condition here. If there's only 1 response, the expression `cmd.Responses[1:]` will return an empty slice (https://play.golang.org/p/ex5aHkdBWJ).